actors.md

```diff
@@ -265,14 +270,31 @@ func SlashConsensusFault(block1, block2 BlockHeader) {
 	self.Miners.Remove(miner)
 	self.UpdateStorage(-1 * miner.Power)
 
-	// Now delete the miner (maybe this is a bit harsh, but i'm okay with it for now)
+	// Remove the actor's state from the network entirely.
 	miner.SelfDestruct()
 }
 ```
 
+### SlashStorageFault
+
+> SlashStorageFault penalizes a storage miner for not submitting a PoSt within the required [time window](#TODO-link-to-faulty-submission). This may be called by anyone who detects the faulty behavior. The slashed miner loses all of their staked collateral and all of their power, and as a result, is no longer a candidate leader for extending the chain. See [faults](faults.md).
```

Comment:
its worth linking directly to the unreported storage fault slashing section that this is for.

Also, what is 'TODO-link-to-faulty-submission' supposed to link to?

Reply:
1. Linkified.
1. I don't know, I copied this from `storage-market.md`. I'll remove the TODO if you don't know what it's standing for.